core/approval/service.go

```diff
@@ -16,6 +16,8 @@ import (
 type repository interface {
 	BulkInsert([]*domain.Approval) error
 	ListApprovals(*domain.ListApprovalsFilter) ([]*domain.Approval, error)
+	AddApprover(*domain.Approver) error
+	DeleteApprover(approvalID, email string) error
```

Comment:
Shouldn't the func signature be the same for these two repository methods?

Reply:
I assume AddApprover is similar to CreateApprover, so that's why I put the struct as a parameter there, and for the deleteApprover, deletion usually only requires the identifier(s)